automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/rstudio/RStudioSpec.scala

```diff
@@ -134,7 +132,8 @@ class RStudioSpec extends RuntimeFixtureSpec with NotebookTestUtils with RStudio
     }
 
     // Note this test should be last because the test infrastructure does not close the shiny app
-    "should launch an RShiny app" in { runtimeFixture =>
+    // TODO: can this test be removed? how important is an rshiny app?
```

Comment:
Not super sure if we need this from a product perspective, and there's a lot of code that exists to support it. It's the only thing keeping this spec's dependency on selenium around...

Reply:
Ohhh that is a good question, I did not even know that we supported RShiny apps, but I don't see how this test does that really. It is just another R package to install so it feels like a test that should be on terra-docker anyways https://shiny.posit.co/r/getstarted/shiny-basics/lesson1/index.html

Reply:
I would love to remove this and the other ~5 files that support it 😄 . @rtitle do you have any input here?